repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -418,6 +444,102 @@ private void traverseEdgesOnDemand(AtlasVertex datasetVertex, boolean isInput, i
         }
     }
 
+    private void traverseEdgesOnDemand(AtlasVertex baseVertex, AtlasLineageListContext lineageListContext, AtlasLineageListInfo ret) throws AtlasBaseException {
```

Comment:
rename to something else so there's confusion b/w graph and list API

Reply:
Resolved